src/actions/SearchActions.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import request from 'superagent'
+import SearchConstants from '../constants/SearchConstants.js'
+
+export const requestCourses = (course: string) => ({
+  type: SearchConstants.REQUEST_COURSES,
+  query: course,
```

Comment:
I'm not sure what you mean – the key here corresponds to the key in the Redux state. I think that makes sense and I'm okay keeping it that way